java/src/main/java/com/logicalclocks/hsfs/engine/FeatureGroupEngine.java

```diff
@@ -56,9 +60,8 @@
    * @throws FeatureStoreException
    * @throws IOException
    */
-  public void saveFeatureGroup(FeatureGroup featureGroup, Dataset<Row> dataset,
-                               List<String> primaryKeys, List<String> partitionKeys,
-                               Storage storage, Map<String, String> writeOptions)
+  public void saveFeatureGroup(FeatureGroup featureGroup, Dataset<Row> dataset, List<String> primaryKeys,
```

Comment:
What happened to the `precombine` flat of the feature object? why don't we set it here?

Reply:
For now I decided to omit as you mention that needed to be discussed. But adding will not be problem

Reply:
ok